docs/api/breadcrumbs.md

```diff
@@ -51,6 +51,40 @@ import CustomSeparators from '@site/static/usage/breadcrumbs/icons/custom-separa
 
 <CustomSeparators />
 
+## Collapsing Items
+
+### Max Items
+
+If there are more items than the value of `max-items`, the breadcrumbs will be collapsed. By default, only the first and last items will be shown.
+
+import MaxItems from '@site/static/usage/breadcrumbs/collapsing-items/max-items/index.md';
+
+<MaxItems />
+
+### Items Before or After Collapse
+
+Once the items are collapsed, the number of items to show can be controlled by the `items-before-collapse` and `items-after-collase` properties.
```

Comment:
```suggestion
Once the items are collapsed, the number of items to show can be controlled by the `itemsBeforeCollapse` and `itemsAfterCollapse` properties.
```

We typically refer to the property names using camel case as that is how they are documented in the "Properties" section.